lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -163,6 +120,9 @@ export default function getRecipeInterface(
                 helpers,
                 userId,
                 disableAntiCSRF,
+                useDynamicAccessTokenSigningKey !== undefined
+                    ? useDynamicAccessTokenSigningKey
+                    : config.useDynamicAccessTokenSigningKey === false,
```

Comment:
```suggestion
                useDynamicAccessTokenSigningKey !== undefined
                    ? !useDynamicAccessTokenSigningKey
                    : config.useDynamicAccessTokenSigningKey === false,
```

Reply:
Should we change createNewSession to take useStaticKeys with default to false instead of using useDynamicAccessTokenSigningKey? Cause the core too takes useStaticKeys

Reply:
I think these were the names In the ADR. I'm open to changing them, but defaulting to true seemed more natural.

Reply:
Agreed about the defaulting to true, so then we should change the core's API - this inconsistency is not necessary and causes confusion in the long run